.travis.yml

```diff
@@ -71,19 +71,25 @@ install:
   - phpenv config-add drupal.php.ini
   - phpenv rehash
 
-  # Build the Lightning code base.
+  # We need Phing to execute our build steps.
   - composer global require phing/phing
-  - phing build -Dversion=$VERSION -Dbranch=$TRAVIS_COMMIT
-
-  # Install Lightning.
-  - phing install -Ddb.database=drupal -Dversion=$VERSION
-  # Execute database updates if we're not building from HEAD.
-  - if [ $VERSION != "HEAD" ]; then phing update; fi
+  # Build the Lightning code base.
+  - composer install
+  # Install Lightning cleanly so that settings.php will be created properly.
+  - phing install -Ddb.database=drupal
+  - cd docroot
+  # If there is a database fixture for the version being tested, recreate the
+  # database from it. We need to destroy the existing database in order to
+  # prevent table collisions.
+  - FIXTURE=$TRAVIS_BUILD_DIR/tests/fixtures/$VERSION.sql.bz2
```

Comment:
I'd be nice if we had a standard way of generating these databases. Maybe just a phing target that does a clean install, enables lightning_preview and does a drush sql-dump that generates a file with the proper name and ext. I see this simple step getting screwed up at least once if we don't.